src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const makeSum = (a) => {
+    if (typeof a === 'number') {
+      sum += a;
+
+      return makeSum;
+    }
```

Comment:
The current implementation checks if the argument 'a' is a number before adding it to 'sum'. According to the task description, there is no need to check the argument type, so this check can be removed.